assets/js/components/Loadpoint.vue

```diff
@@ -138,28 +171,59 @@ import formatter from "../mixins/formatter";
 
 export default {
 	name: "Loadpoint",
-	props: ["state", "id", "pv", "multi"],
+	props: {
+		id: Number,
+		multi: Boolean,
+		pvConfigured: Boolean,
+
+		// main
+		title: String,
+		mode: String,
+		targetSoC: Number,
+		socLevels: Array,
+		remoteDisabled: Boolean,
+		remoteDisabledSource: String,
+		chargeDuration: Number,
+		charging: Boolean,
+
+		// vehicle
+		connected: Boolean,
+		// charging: Boolean,
+		socTitle: String,
+		socCharge: Number,
+		minSoC: Number,
+
+		// details
+		chargePower: Number,
+		chargedEnergy: Number,
+		// chargeDuration: Number,
+		soc: Boolean,
+		climater: String,
+		range: Number,
+		chargeEstimate: Number,
```

Comment:
Siehe oben. Hier nur die Props direkt rein, die auch im Template auftauchen. Gruppen die nur weitergegeben werden dann als Objekt. Das hat natürlich zur Folge, dass man das Datenmodell in der API umbauen muss, aber das macht auch da sicher aus Lesbarkeitsgründen Sinn.